### PR TITLE
CSS-syntax - Tyler Stahl

### PR DESCRIPTION
**Pull request by TyStahl**
Screenshot css-syntax

### CODE REVIEW

File: css-syntax/styles.css [hunk 1]
```diff
@@ -0,0 +1,28 @@
+body {
+  color: 090302;
+  font-family: sans-serif;
+}
+
+h1 {
+  padding-left: 16px;
+  padding-right: 16px;
+  border-radius: 6px;
+  margin-bottom: 16px;
```
Comment (shawnkost) [inline css-syntax/styles.css]: This should be on the `article`

File: css-syntax/styles.css [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+}
```
Comment (shawnkost) [inline css-syntax/styles.css]: Missing the correct styles for your `h1`

**Review by shawnkost — CHANGES_REQUESTED** [review]
Some changes here Tyler! Make sure to go back and reread the instructions. Some of your styles are being applied in the wrong places

**Review by shawnkost — APPROVED** [review]
👍🏻

**Review by TyStahl — COMMENTED** [review]
Screenshot css-syntax

**Pull request merged**